irc/client.go

```diff
@@ -578,10 +599,10 @@ func (session *Session) playResume() {
 		}
 		for _, fSession := range friend.Sessions() {
 			if fSession.capabilities.Has(caps.Resume) {
-				if timestamp.IsZero() {
-					fSession.Send(nil, oldNickmask, "RESUMED", hostname)
-				} else {
+				if session.resumeDetails.HistoryIncomplete {
```

Comment:
probably makes sense to `if session.resumeDetails.HistoryIncomplete && !session.resumeDetails.Timestamp.IsZero()` here to make sure we aren't sending a zero-time that could make 'em display _"40 years of history lost"_ or something? we can't rely on `timestamp` for the `IsZero` check because `timestampString` is generated based off `resumeDetails.Timestamp` rather than `timestamp`.